templates/workloads/partials/cf/container-definition-base-properties.yml

```diff
@@ -0,0 +1,13 @@
+ContainerDefinitions:
```

Comment:
I think we should take either one of these routes:
1. Transform this file to `container-definitions.yml` and include `sidecars` here.
2. Or alternatively rename this file to `workload-container.yml` and move `ContainerDefinitions` outside of this file so that `sidecars.yml` and this file ends up being individual elements in the list

Reply:
I've done 2) in the latest commit.